ui/src/callWithTelemetryAndErrorHandling.ts

```diff
@@ -20,15 +20,9 @@ function initContext(): [number, IActionContext] {
     const context: IActionContext = {
         telemetry: {
             properties: {
-                isActivationEvent: 'false',
-                cancelStep: '',
-                result: 'Succeeded',
-                stack: '',
-                error: '',
-                errorMessage: ''
+                result: 'Succeeded'
```

Comment:
Removing this does have one disadvantage.  If we don't send this by default, we won't be able to register them until they're used.  I.e., we won't get actual stacks, error messages or cancel steps coming through our telemetry for each event until a) a user actually hits an error, and b) we categorize it after noticing it's missing and then wait for another hit.

Maybe we should add a new env var that's turned on when running tests which causes these to be set to empty strings only then.  I've never liked sending these in all events, but I don't like not registering everything up front, either.

Reply:
> I've never liked sending these in all events, but I don't like not registering everything up front, either.

I guess I don't have a strong preference. Want me to revert this part of the PR for now?